src/client/opamAdminCommand.ml

```diff
@@ -866,6 +866,8 @@ let state_selection_arg cli =
      $(b,environment) (this calls the solver and may be more costly; \
      a package depending on an unavailable one may be available, but \
      is never installable)";
+    OpamArg.cli_from OpamArg.cli2_3, OpamListCommand.Latests_only,
```

Comment:
We should add also a test on opam admin test, as the option is also added to it.

Reply:
done